src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -119,12 +119,13 @@ bipyramid(P::Polyhedron{T}, z::Number, z_prime::FieldElem) where {T<:scalar_type
   bipyramid(P, parent(z_prime)(z), z_prime)
 
 @doc raw"""
-    normal_cone(P::Polyhedron, i::Int64)
+    normal_cone(P::Polyhedron, F::Vector{Int64}; outer::Bool)
 
-Construct the normal cone to `P` at the `i`-th vertex of `P`.
+Construct the normal cone to `P` at the corresponding to `F` vertex indices which are not contained in the far face `P`.
```

Comment:
I do not understand this sentence. Do you maybe mean something along the lines of
```suggestion
Construct the normal cone to `P` at the face corresponding to vertex indices in `F`.
```
? In this first sentence it is probably ok to omit mentioning the far face, this can be done further down.

Reply:
Oh what did I wrote there actually, it's normal that it's not understandable. The thing you have written is exactly what I wanted to say. I also tried to rewrite it using polymake documentation.

Reply:
If you agree with Lars proposal, you could just apply the suggestion (by using the button for it here in the Web UI, or just pushing the changes into your branch)

Reply:
But I thought it might mess up when there are several things to change instead. But I will try it out next time!